packages/pwa-kit-react-sdk/package.json

```diff
@@ -43,6 +43,7 @@
     "@loadable/babel-plugin": "^5.13.2",
     "@loadable/server": "^5.15.0",
     "@loadable/webpack-plugin": "^5.15.0",
+    "@tanstack/react-query": "^4.0.10",
```

Comment:
It doesn't look like we can enforce the version of `react-query` that is used in the template code without making a breaking change. That change being making `react-query` a peer dependency. 

We've in used optional peer dependencies in other modules, but optional peer deps aren't supported in some of the lower versions of npm that we support. 

Do you think it's ok to just leave this as is, and if someone was to upgrade the version of their `react-query` in their project we simply make no promises on it working with the SDK?

Reply:
Honestly, I'd use the optional peer dep. We've used it before. I think it's just ignored if it's not supported by an NPM version, right?

Reply:
So for node 7 which doesn't know anything about peerDependenciesMeta optional, if would just think that react-query is a peer dep. Which is a breaking change.